### PR TITLE
Count Prefix Using Trie

### PR DESCRIPTION
**Pull request by Pranshu321**
## Related Issue
- #1316 

## Proposed Changes
- Cpp file added
- Readme File Added

## Additional Info
- In, This I have explained how we can count the prefix of word or we say the words that start with that prefix and return the count of that.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

In this, I have deeply explained how we approach the question of what should be the node structure and also written well commented code which make more understanding of topic.

Prefix count using trie

#1316 

Any GCC Compiler

- [x] New feature (non-breaking change which adds functionality)

## Output Screenshots
| Screenshot #1      |
| ----------- | 
| Code Output | 
image

### CODE REVIEW

File: T-Trie/Count_Prefix.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+int main()
+{
+    Trie T;
+    string s[] = {"the", "this", "tear", "cute", "monkey"}; // The list
+    for (auto i : s)
+    {
+        T.insert(i);
+    }
+    string check = "th"; // The prefix which we have to check.
```
Comment (siddhi-244) [inline T-Trie/Count_Prefix.cpp:85]: Try to take this as input !

Comment (Pranshu321) [reply T-Trie/Count_Prefix.cpp]: Done @siddhi-244 , Please review it

**Comment by Pranshu321** [comment]
> Can you check the approach once?

And yes, I have checked the approach and don't worry mam @siddhi-244

**Comment by Pranshu321** [comment]
> > Can you check the approach once?
> 
> And yes, I have checked the approach and don't worry mam @siddhi-244

@siddhi-244 please review this

**Comment by Pranshu321** [comment]
> Add table of contents in the readme.

@siddhi-244 added

**Review by siddhi-244 — APPROVED** [review]
LGTM

File: T-Trie/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<hr>
+
```
Comment (Lakhankumawat) [inline T-Trie/README.md:2]: You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

File: T-Trie/README.md (hunk 2 above)
Comment (Pranshu321) [reply T-Trie/README.md]: Now check sir , the guidelines followed

**Review by Lakhankumawat — APPROVED** [review]
## LGTM 💎🚩

**Pull request merged**